lib/pwnlib/reg_sort.rb

```diff
@@ -190,13 +176,29 @@ def regsort(in_out, all_regs, tmp: nil, xchg: true, randomize: nil)
 
       private
 
+      # Walk down the assignment list of a register,
+      # return the path walked if it is encountered again.
+      #
+      # @return [Array<String>]
```

Comment:
Either add @param doc, or remove docs and just use single-line doc as in `depends_on_cycle` since this method is private?